### PR TITLE
Enable import additional resources when creating source

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

File: client/src/components/newSource/index.tsx [hunk 1]
```diff
@@ -199,8 +253,9 @@ const NewSourceView = (): React.ReactElement => {
         if (!templateExists) {
           await createTemplate();
         }
-        // Create new source in Graphql
-        await createSource();
+        // Create new source and upload bundle in parallel
+        await Promise.all([createSource(), uploadFhirBundle()]);
+        // await Promise.all([uploadFhirBundle()]);
```
Comment (simonvadee) [inline client/src/components/newSource/index.tsx:258]: remove comment

Comment (simonvadee) [reply client/src/components/newSource/index.tsx:258]: upload the bundle after creating the source

File: client/src/components/newSource/index.tsx [hunk 2]
```diff
@@ -226,7 +195,28 @@ const NewSourceView = (): React.ReactElement => {
...
+        )
+          reject(
+            new Error(
+              `Source created but failed while trying to upload a bundle with 
+              resource which type is not
+              StructureDefinition, ConceptMap, nor CodeSystem`
+            )
+          );
+        // Upload the bundle
+        await axios.post(`${FHIR_API_URL}/upload-bundle`, bundleJson);
```
Comment (simonvadee) [inline client/src/components/newSource/index.tsx:219]: what's the maximum size that the API accepts ? maybe we should mention it "your bundle cannot exceed 1mb" or something)

Comment (Jasopaum) [reply client/src/components/newSource/index.tsx:219]: No idea, not sure how this is handle by flask but I'll have a look

**Pull request merged**